### PR TITLE
Transpile esm output too

### PR DESCRIPTION
**Pull request by guillaumewuip**

### CODE REVIEW

File: scripts/configs.js [hunk 1]
```diff
@@ -99,10 +98,8 @@ function genConfig(options) {
 
   config.plugins.push(replace(vars))
 
-  if (options.transpile !== false) {
-    config.plugins.push(babel())
-    config.plugins.push(buble())
-  }
+  config.plugins.push(babel())
+  config.plugins.push(buble())
```
Comment (guillaumewuip) [inline scripts/configs.js:102]: I'm not sure about Bublé here. Why to we need it in addition to babel ? In my mind `@babel/preset-env` was sufficient to output ES5 code

**Comment by catsintents** [comment]
We're unable to upgrade fuse.js without these changes, could this be merged or at least looked at again?

**Comment by guillaumewuip** [comment]
@krisk this is the PR following #381

**Comment by mcstover** [comment]
+1 I'd also like to see if this helps the IE compilation issues.

In the meantime, we are importing the `common` build instead which seems to be working ok. Slightly larger bundle size, but functional.

**Comment by JulianH** [comment]
+1 this will fix the IE11 problem, verified it just with https://github.com/JulianH/Fuse/tree/fix-babelrc-for-ie11-object-spread

**Comment by mjoyce91** [comment]
Still an issue in the 6.0.0 release. Would like to upgrade, but can't until the IE compatibility is addressed.

**Comment by revolunet** [comment]
thanks for the PR, this will also address issues on firefox 52.

**Comment by willviles** [comment]
Not transpiling object spreads properly is breaking our IE11 & Microsoft Edge (non-Chromium) builds too. Would be great to get this merged and released!

**Comment by silentHoo** [comment]
Would be nice to have this fix in the next release.

**Pull request closed without merging**